### DISCUSSION TITLE
Use `canonical_version` to defer Google etc. indexing old docs, but how?

### DISCUSSION THREAD

**Discussion by jnummelin** [jimporter/mike#179] [Q&A] [answered]
In https://github.com/jimporter/mike/issues/159#issuecomment-1597517697 `canonical_version` was mentioned to tell Google to prefer the latest version.

Unfortunately I see no documentation on it, would anyone be able to provide some examples how that is supposed to be used?

In our case we produce docs for each version we release. We've ended up in situation where Google sometimes takes the user to VERY old version of the docs. Essentially we'd like to Google always prefer only the latest stable release. Can we do that with `canonical_version`?

**Comment by jimporter** [comment, marked as answer]
There's documentation on this in the README: https://github.com/jimporter/mike#configuration

**Reply by jnummelin** [reply]
Oh, I must be really getting blind as I completely missed that. 🤦

**Comment by jnummelin** [comment]
Is there any easy way to apply `canonical_version` to already published versions?

**Reply by jimporter** [reply]
You'll have to deploy updates to those versions as usual.
